avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -1107,7 +1107,7 @@ function Set-UsageExamplesSection {
 
     # Process content
     $SectionContent = [System.Collections.ArrayList]@(
-        'The following module usage examples are retrieved from the content of the files hosted in the module''s `tests` folder.',
+        "The following section provides usage examples for the module, which were used to validate and deploy the module successfully. For a full reference, please review the module's test folder in its repository.",
```

Comment:
👻

Reply:
Not a ghost @eriqua. If we don't have this change in (before #546) is merged, readme's would become outdated again.

Reply:
gone it is